coldfront/config/defaults.py

```diff
@@ -0,0 +1,155 @@
+"""Storage of default choices for attribute and status values."""
+
+### Allocation ###
+
+ALLOCATION_DEFAULTS = {
+    'attrtypes': (
+        'Date',
+        'Float',
+        'Int',
+        'Text',
+        'Yes/No',
+        'No',
+        'Attribute Expanded Text'
+    ),
+    'statuschoices': (
```

Comment:
Was considering adding comments to this file that indicate which default options are required for Coldfront to function - e.g., the "Active" status choice is so ubiquitous in the codebase that removing it as a default would require a lot of changes.